components/Tables/ContributorsTable.tsx

```diff
@@ -121,7 +122,12 @@ const defaultColumns = (repository: string) => [
     ),
   }),
   contributorsColumnHelper.accessor("oscr", {
-    header: "Rating",
+    header: () => (
+      <div className="flex gap-2 w-fit items-center hover:bg-slate-50 rounded-md px-2 py-1 cursor-pointer">
+        <p>OSCR Rating</p>
+        <InfoTooltip information="OSCR evaluates the engagement and impact of contributors across the entire open source ecosystem." />
+      </div>
+    ),
     enableSorting: true,
     enableGlobalFilter: false,
     cell: (info) => <OscrPill rating={info.row.original.oscr ?? 0} />,
```

Comment:
For the `<OscrPill />` components, add the required `hideRating` prop and set it's value to whether or not the user is logged in or not.